ci/test_cpp.sh

```diff
@@ -35,7 +35,7 @@ EXITCODE=0
 trap "EXITCODE=1" ERR
 set +e
 
-ctest -j20 --schedule-random --output-on-failure --no-tests=error
+ctest -j20 --schedule-random --repeat pass:3 --output-on-failure --no-tests=error
```

Comment:
Attempting to fix: https://cmake.org/cmake/help/latest/manual/ctest.1.html#cmdoption-ctest-repeat
```suggestion
ctest -j20 --schedule-random --repeat until-pass:3 --output-on-failure --no-tests=error
```